src/crawler/storybook.ts

```diff
@@ -357,6 +358,7 @@ export const generateStorybookShotItems = (
           story.parameters?.lostpixel?.waitBeforeScreenshot ??
           config.waitBeforeScreenshot,
         mask: [...(mask ?? []), ...(story.parameters?.lostpixel?.mask ?? [])],
+        elementLocator: story.parameters?.lostpixel?.elementLocator,
```

Comment:
on shotItem level I would also try to pick from the shot params and if not existent than from the overall storybook config. can't see it right now, maybe I am just blind, point me in the right direction if you have accounted for that already.